client/src/components/RelatedProducts/RP.css

```diff
@@ -0,0 +1,54 @@
+
+.related-products-frame {
+  display: grid;
+  height: 375px;
+  border:  1px solid rgb(223, 30, 30);
```

Comment:
Andrew - 

Looks like you've done a lot of work with CSS !   

The only concern I have is if the rest of us are using Flexbox if there will be any rendering issues with your component using grid as well.  It doesn't seem Likely because our components have their own styling, but I thought I should bring it up.

Reply:
Oh I honestly did not think about that. But sure we can discuss how maybe I can change it if we need.